deployments/config/local/1001-polygon-zkevm.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Polygon ZkEVM Local",
+    "chain_id": 1001,
+    "l1_messenger": [
+        "PolygonZkEVMMessenger",
+        ${native_bridge},
+        "${resolver}",
+        1
+    ],
+    "l2_messenger": [
+        "PolygonZkEVMMessenger",
+        ${native_bridge},
+        "${request_manager}",
```

Comment:
The addresses should always be strings, like the request manager example here. So you are missing quotes here and the adjustment of types in the Python part is not necessary.

Reply:
the problem is coming from chain direction being integer and i got validation error as tuple only expects str

Reply:
that is why i have a change in config and for native bridge usage here also written in optimism-local-template.json and arbitrum-local-template.json

Reply:
> the problem is coming from chain direction (0, 1) being integer and i got validation error as tuple only expects str

I don't understand. Why can't we have
```
        "PolygonZkEVMMessenger",
        "${native_bridge}",
        "${request_manager}",
```
?

Reply:
now works